pkg/test/framework/components/istio/config.go

```diff
@@ -16,14 +16,20 @@ package istio
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
 	"strings"
 	"time"
 
+	"github.com/mitchellh/go-homedir"
```

Comment:
What is this? Don't golang has os.UserHomeDir that could be used since 1.12? ;-)

Reply:
I am reverting a PR, I didn't touch any code here...